src/src.pro

```diff
@@ -463,4 +463,6 @@ DISTFILES += \
     ../cmake/FindPCRE.cmake \
     ../cmake/FindYAJL.cmake \
     ../cmake/FindZIP.cmake \
-    .clang-format
+    .clang-format \
+    ../../../../Desktop/copy.png \
```

Comment:
Yeah, if you are going to include icons grab them from the Oxygen 48x48 set that most Linux distributions have lying around - if you do not have such things let me know and I'll inject them in a PR to your Repo @dicene ...